.github/workflows/ci.yml

```diff
@@ -0,0 +1,13 @@
+name: CI
```

Comment:
Add all the other module-standardiser .yml files like dispatch-ci.yml, auto-tag.yml etc

Reply:
That will happen when I run module standardiser, which will be a separate PR since some of those scripts rely on specific files existing in the repo first.